SU-HSDO/suhumsci
# SHS-107: Implement Gin admin theme

# DO NOT DELETE: Deployed to [aquia dev link](https://hs-colorful-dev.stanford.edu/) for previews and testing

## Summary
- Installs and configures Gin as a new admin theme
- Provides a basic custom color palette implementations that can be extended for each site

## Need Review By (Date)
TBD

## Urgency
low

## Steps to Test
1. Run a rebuild (lando rebuild -y) and sync (lando blt drupal:sync --site=SITENAME) with a random set of sites (2-3 is a good test). 
2. Verify there are no errors reported during each blt sync command run
3. Logging into the admin area and going to `/admin/appearance` shows that Claro, Gin, and SU HumSci Gin Admin are installed. "SU HumSci Gin Admin" is set at the Administration Theme.
4. Seven and HumSci Admin are an uninstalled theme

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
